build/reports/license/license-dependency.xml

```diff
@@ -0,0 +1,198 @@
+<licenses>
+  <license name='CC0 1.0 Universal' url='http://creativecommons.org/publicdomain/zero/1.0/'>
+    <dependency>jsr166e-1.0.0.jar</dependency>
+  </license>
+  <license name='No license found'>
+    <dependency>base-0.0.114.jar</dependency>
+    <dependency>application-action-0.0.89.jar</dependency>
+    <dependency>zipkin-cassandra-core-1.23.4-SNAPSHOT.jar</dependency>
```

Comment:
guess we need to do something to fix this? suppose changing the jar plugin to add a license file?